Source/Core/DolphinQt/Config/MouseSettingsWidget.cpp

```diff
@@ -0,0 +1,227 @@
+#include "DolphinQt/Config/MouseSettingsWidget.h"
+#ifdef _WIN32
+#include <Windows.h>
+#endif
+
+#include <memory>
+
+#include <QApplication>
+#include <QCheckBox>
+#include <QDoubleSpinBox>
+#include <QLabel>
+#include <QVBoxLayout>
+
+#include "Core/Config/MouseSettings.h"
+#include "DolphinQt/QtUtils/MultiClickSignalButton.h"
+#include "InputCommon/ControllerInterface/OctagonalMouseJail.h"
+
+MouseSettingsWidget::MouseSettingsWidget(QWidget* parent) : QGroupBox(parent)
+{
+  CreateLayout();
+  LoadSettings();
+  ConnectWidgets();
+}
+
+void MouseSettingsWidget::CreateLayout()
+{
+  QVBoxLayout* primary_layout = new QVBoxLayout{};
+
+  QHBoxLayout* sensitivity_layout = CreateSensitivityLayout();
+  QHBoxLayout* snapping_distance_layout = CreateSnappingDistanceLayout();
+  QHBoxLayout* center_mouse_key_layout = CreateCenterMouseKeyLayout();
+
+  QHBoxLayout* octagonal_mouse_jail_is_enabled = CreateOctagonalMouseJailIsEnabledLayout();
+
+  primary_layout->addLayout(octagonal_mouse_jail_is_enabled);
+  primary_layout->addLayout(sensitivity_layout);
+  primary_layout->addLayout(snapping_distance_layout);
+  primary_layout->addLayout(center_mouse_key_layout);
+
+  this->setLayout(primary_layout);
+}
+
+void MouseSettingsWidget::ConnectWidgets()
+{
+  // I'm not sure what QOverland does in this, but it was in the example in the
+  // qt documentation and it doesn't work without it.
```

Comment:
It's so that if there are multiple `valueChanged()` functions, you get the one that takes a `double` parameter. You can remove this comment.